src/utils/object.utils.ts

```diff
@@ -26,3 +26,34 @@ export const hashObject = async <T extends object>(object: T) => {
     const hashedString = hashArray.map((b) => ("00" + b.toString(16)).slice(-2)).join("");
     return hashedString;
 };
+
+type DeepPartial<T> = {
```

Comment:
maybe it would be better to create some types folded or file and store it there, coz it can be reused later

Reply:
you can also copy DeepPartial from deriv-app

Reply:
This is not the same. Let me rename to avoid confusion if that helps. Also this is generic type that needn't be exposed. So I think there is no need to create a types file and share it